index.js

```diff
@@ -184,6 +207,12 @@ var Umzug = module.exports = redefine.Class({
     }
   },
 
+  log: function() {
+    if (this.options.logging) {
+      this.options.logging.apply(null, [].slice.call(arguments));
```

Comment:
What do you expect to happen when the logging method is called with multiple arguments?